salt/top.sls

```diff
@@ -17,6 +17,8 @@ base:
     - tls
     - rsyslog
     - datadog
+    - datadog.synthetics
+    - secrets.datadog
```

Comment:
is this the right place for this? it felt wrong but it needed them to be available